permissions/plugin-downstream-ext.yml

```
@@ -6,3 +6,5 @@ issues:
 paths:
   - "org/jenkins-ci/plugins/downstream-ext"
 developers: []
```

I think that you want to be a developer here as well
```suggestion
developers:
  - "notmyfault"
```